draft-ietf-quic-transport.md

```diff
@@ -1257,6 +1268,27 @@ frames), the receiving peer MAY send an ACK frame after a reasonable number
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+## Special Considerations for Path MTU Discovery 
+
+Traditional ICMP-based path MTU discovery in IPv4 ({{!RFC1191}} is potentially
+vulnerable to off-path attacks that succesfully guess the IP/port 4-tuple and
+reduce the MTU to a bandwidth-inefficient value. TCP connections mitigate this
+risk by using the (at minimum) 8 bytes of transport header echoed in the ICMP
+message to validate the TCP sequence number as valid for the current
+connection. However, as QUIC operates over UDP, in IPv4 the echoed information
+may consist only of the IP and UDP headers, which usually has insufficient
+entropy to mitigate off-path attacks.
+
+As a result, endpoints that implement PMTUD in IPv4 SHOULD take steps to mitigate
+this risk. For instance, an application may:
+
+* Set the IPv4 Don't Fragment (DF) bit on a small number of packets per RTT, so
+that most invalid ICMP messages arrive when there are no DF packets
+outstanding.
+
+* Store additional information from the IP or UDP headers (for example, the IP
+ID or UDP checksum) to further authenticate incoming Packet Too Big messages.
```

Comment:
Maybe just "store the first IP and UDP headers from DF packets" to further authenticate [...]"